### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by Simbul**
@joshuafleck @martinbtt @cpoo22 please review.

This is us adding Rubocop to this repo as part of a Level Up session.

We only have addressed two minor linting issues. To complete enabling Rubocop in this repo, we need to move all checks in `.rubocop_todo.yml` to `.rubocop.yml` one by one, enabling them as we go.

### CODE REVIEW

**Comment by martinbtt** [comment]
Looking good! 

Don't forget the CI check: https://github.com/simplybusiness/continuous_cover/blob/rubocop_levelup/script/cibuild

Then on semaphore (example)
https://semaphoreci.com/simplybusiness/continuous_cover/settings

```
[ -f ./script/cibuild ] && sh ./script/cibuild || echo "Skipping. No cibuild script to run"
```

**Comment by Simbul** [comment]
@martinbtt I have added the CI script: Rubocop is now running on Semaphore.

I don't have time at the moment to go through all the disabled rules, but this should at least provide a foothold for whoever wants to go ahead and work on one or three :wink: 

@martinbtt @joshuafleck can I haz signoff please? :smiley:

File: flip_fab.gemspec [hunk 1]
```diff
@@ -23,4 +23,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec'
   spec.add_development_dependency 'rutabaga'
   spec.add_development_dependency 'timecop'
+
```
Comment (joshuafleck) [inline flip_fab.gemspec]: extra newline

**Comment by joshuafleck** [comment]
signed-off-by: @joshuafleck

File: flip_fab.gemspec (hunk 1 above)
Comment (Simbul) [inline flip_fab.gemspec]: In a PR to introduce a linter tool. Such irony :smile: 
Fixed in 3b90e49.

**Pull request merged**